laghos_rom.hpp

```diff
@@ -183,30 +186,45 @@ class ROM_Sampler
 
         if (offsetInit)
         {
-            //std::string path_init = (parameterID >= 0) ? "run/ROMoffset/param" + std::to_string(parameterID) + "_init" : "run/ROMoffset/init"; // TODO: Tony PR77
-            std::string path_init = "run/ROMoffset/init";
             initX = new CAROM::Vector(tH1size, true);
             initV = new CAROM::Vector(tH1size, true);
             initE = new CAROM::Vector(tL2size, true);
             Xdiff.SetSize(tH1size);
             Ediff.SetSize(tL2size);
 
-            for (int i=0; i<tH1size; ++i)
-            {
-                (*initX)(i) = X[i];
-            }
-            initX->write(path_init + "X" + std::to_string(window));
+            //std::string path_init = (parameterID >= 0) ? "run/ROMoffset/param" + std::to_string(parameterID) + "_init" : "run/ROMoffset/init"; // TODO: Tony PR77
+            std::string path_init = "run/ROMoffset/init";
```

Comment:
I pushed some changes to rom-dev that will affect this line. Can you change it to 

std::string path_init = basename + "/ROMoffset/init";